release/chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,27 @@
+
+$ErrorActionPreference = 'Stop';
+
+$packageName= 'Streamlink'
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$url        = "https://github.com/streamlink/streamlink/releases/download/$env:chocolateyPackageVersion/streamlink-$env:chocolateyPackageVersion.exe"
```

Comment:
Have you considered converting this into what we refer to as an embedded package?  That is where you would put the exe directly within the Chocolatey nupkg file, and install from there, rather than downloading the exe during the installation.  This would mean that the package is completely contained, and can be installed anywhere without the need to reach back out to the internet to grab the installer.